src/submitter.py

```diff
@@ -423,23 +441,36 @@ def handler(event, context):
             continue
 
         LOGGER.debug("saw crash id: %s in %s", crash_id, bucket)
-
-        # Throttle crashes
```

Comment:
I rearranged `handler` to support multiple destinations and throttles.

First, we pull crash ids from records.

Then we do a loop over each crash id and throttle it to figure out which places to submit the crash to.

Then we build the payload and loop over the submit destinations to send it to whatever places we want to send it to.

This keeps the amount of work the handler does roughly the same modulo one `random.randint` call and possibly one `requests.post` per destination.